src/magic_search.js

```diff
@@ -3,7 +3,7 @@
  * @requires AngularJS
  *
  */
-angular.module('MagicSearch', [])
+angular.module('MagicSearch', $(document).foundation ? [] : ['ui.bootstrap'])
```

Comment:
It looks like foundation isn't initialized early enough to help here. I'll see what else might work.